projects/collection-editor-library/src/lib/components/fancy-tree/fancy-tree.component.ts

```diff
@@ -255,12 +255,14 @@ export class FancyTreeComponent implements OnInit, AfterViewInit, OnDestroy {
     // tslint:disable-next-line:max-line-length
     this.visibility.addSibling = ((node.folder === true) && (!node.data.root) && !((node.getLevel() - 1) > this.config.maxDepth)) ? true : false;
     if (nodeLevel === 0) {
-      this.visibility.addFromLibrary = _.isEmpty(_.get(this.config, 'children')) ? false : true;
-      this.visibility.createNew = _.isEmpty(_.get(this.config, 'children')) ? false : true;
+      this.visibility.addFromLibrary = _.isEmpty(_.get(this.config, 'children')) || _.get(this.config, 'enableQuestionCreation') === false ? false : true;
+      this.visibility.createNew = _.isEmpty(_.get(this.config, 'children')) || _.get(this.config, 'enableQuestionCreation') === false ? false : true;
     } else {
       const hierarchylevelData = this.config.hierarchy[`level${nodeLevel}`];
-      this.visibility.addFromLibrary = ((node.folder === false) || _.isEmpty(_.get(hierarchylevelData, 'children'))) ? false : true;
-      this.visibility.createNew = ((node.folder === false) || _.isEmpty(_.get(hierarchylevelData, 'children'))) ? false : true;
+      // tslint:disable-next-line:max-line-length
+      this.visibility.addFromLibrary = ((node.folder === false) || _.isEmpty(_.get(hierarchylevelData, 'children')) || _.get(this.config, 'enableQuestionCreation') === false) ? false : true;
```

Comment:
@bharatkashyap what will happen if enableQuestionCreation is undefined?

Reply:
The condition will evaluate to false (`undefined` === `false` will be `false`) and the value will depend on the originally present conditions, since:
(`x` || false = `x`)